### DISCUSSION TITLE
White balance block and quick capture

### DISCUSSION THREAD

**Discussion by Herculiano** [gopro/labs#510] [General]
Hi,

I'm trying to use this Lab features for diving and I got a question I was not able to solve searching the net.

If I set the white balance lock on, and at the same time I have the quick capture on also.

When I press the capture button, how does it work? Is there time for the camera to measure the conditions, and lock the white balance before starting to record. If the record button what looks the white balance or the recording event it self?

Sorry for my bad English, I hope you understand my doubt.

I would like to know the way it works if I active te both things at the same time.

Thanks in advance for your help.

Juan

**Comment by dnewman-gpsw** [comment]
I just tried it, it works fairly well. I set !MWBLK=1 then attempted several quick capture under different initial lighting conditions and white matched locked 4.5 seconds into the capture. So the QC starts before Labs the code, but the lock still works, if only a little delayed.

**Reply by Herculiano** [reply]
Thanks a lot for your fast answer.

Nice that it works that way.

Do you know if I can use !MWBLK=1 at the same time than !MWIDE=1 in an script? Or using the QR generator set WBLK=1 and WIDE=1?

Trying to learn how to work wit this things and generate a reduced number of QR to laminate then for use underwater. 

My main problem is that I'm running out of time for this as I traveling on a dive trip next Saturday 😅

**Reply by dnewman-gpsw** [reply]
All can be set at the same time.  But there are limits to the command length as the QR codes get harder to read.  You can save batches of commands in loadable scripts, you could have a script !Ldive in another script !Lsurface.  This would keep the QR code you laminated simple.

**Reply by Herculiano** [reply]
This is my intent, to have a QR code for diving with this settings:
GPS Off
WIFI Off
LEDS Rear ON
Beep volume 100%
Voice ctrl OFF
Auto off Never
Back light 100%
Quik Cap ON
LCD off Never
Color flat
Color depth 10 bits
Bit rate High
EV comp -0.5
Sharpness Medium
EIS HS BOOST
WBLK=1
WIDE=1

Then another one for surface with that:
GPS On
WIFI On
Beep volume 100%
Voice ctrl ON
Auto off 5 min
Back light 100%
Quik Cap ON
LCD off 1 min
Color flat
Color depth 10 bits
Bit rate High
EV comp 0EV
Sharpness Medium
EIS HS BOOST
WBLK=0
WIDE=1

Then I want to have other QRs to control the resolution/fps/Lens 

I'm following the Custom exposure curve for Scuba #382 work of **chrisrscuba**

So I will use his macro and have a QR to disable it to make test: I will try to capture the same scene with this curve and with the "normal" one.
Also to disable it when I will use high fps as he made it for 30 and 60 fps only.

A lot of things I would like to test but unfortunately I don't think I will have the chance to generate, print and laminate QRs on the go in Egipt. So I will prepare some of them and lets see how it goes.

Just the ability to change the video resolution and fps its a great thing, planning also to have a couple of QRs to go up and down with the EV

Thanks for your help.

regards form Ramstein

**Comment by dnewman-gpsw** [comment]
EIS HS Boost is likely not the right choice for dive, water crops the FOV, Boost more so.  Better to just HS standard for a greater FOV.

**Comment by dnewman-gpsw** [comment]
In water stabilization is often less of an issue, EIS off for the best FOV.

**Reply by Herculiano** [reply]
Thanks for review my setting, the stabilization setting is what I got form different sources... but your rationale sound logical to me.

I will prepare a second dive QR without it. Test and choose.

**Comment by Herculiano** [comment]
Tested!

I should have tested the WBLK before going to an actual dive. It doesn't work as expected. When you have it set (WBLK=1) and you press the record button it start to capture and a couple of seconds later it start to darken the picture till the maximum. So you end up with captures that are useless, no way to recover the light (or at least I was not able) in post with Da Vinci Resolve.

I don ´t know if this is a problem of interaction with other settings but for me is absolutely useless, even more... it destroy your footage.

Regards from Marsa Alam

**Reply by dnewman-gpsw** [reply]
Which camera and firmware are you running? That was a known issue in older firmware, but I can't reproduce that on HERO11 with the current firmware 2.12.71.  If is happening on the current firmware, please let me know what mode and settings you are running (or share a clip than failed.)

**Reply by ni-73** [reply]
@dnewman-gpsw you need relatively low light to reproduce it using the combination of WBLCK and WIDE. In bright light it does not happen. But in dim light it does. The dimmer it is the more pronounced (exponentially) the effect becomes to the point that results in pitch black video recorded, despite showing much brighter in the preview and while recording with same settings but without WIDE.

@Herculiano Keep WBLCK but remove WIDE and you should be fine. Also while diving you don't need backlit 100%, try 40% to preserve battery. Also, my experience with EIS in dimly lighted scuba environment was awful. Try without it and selectively stabilize at post when needed.

**Reply by dnewman-gpsw** [reply]
@ni-73 That is the lead needed, I wasn't aware of the WIDE and WBLK interaction.  That should be fixable.

**Reply by dnewman-gpsw** [reply]
That was a simple bug, it can be fixed in next Labs release for HERO11 & 12.  In the meantime you will need to choose WIDE or WBLK not both.

**Reply by Herculiano** [reply]
I don't know what happened with my last answer...

Anyway, I'm using the latest firmware 2.12.70

I have the WIDE=1 setting in a QR with other settings so it's not possible to eliminate it easily. I will not use WBLCK. At the moment I'm happy with the auto setting so it's OK.
Next trip I'm going to prepare a QR for each individual setting option and then choose which ones get into the water.

A better planning would have been great.

But anyway happy with the possibility of changing fps and resolution and other things underwater.

A great advantage in compare with other action cameras.
